presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -84,6 +88,32 @@ public void testBooleans()
         assertQueryUsingH2Cte("SELECT COUNT(*) FROM lineitem_ex WHERE ship_by_air is not null AND is_returned = true");
     }
 
+    @Test
+    public void testBytes()
```

Comment:
I'm curious how good is test coverage of this test?

Reply:
line coverage is 82%.
com.facebook.presto.orc.reader.ByteSelectiveStreamReader | 100% (1/1) | 100% (19/19) | 82% (200/243)
-- | -- | -- | --